.azure_pipeline.yml

```diff
@@ -1,26 +1,95 @@
-# Adapted from https://github.com/pandas-dev/pandas/blob/master/azure-pipelines.yml
 jobs:
-
-- template: continuous_integration/azure/windows.yml
-  parameters:
-    name: Windows
-    vmImage: vs2017-win2016
+- job: 'test_loky'
+  strategy:
     matrix:
-      py37_conda:
-        VERSION_PYTHON: '3.7'
-        PACKAGER: 'conda'
-      py36_pip:
-        VERSION_PYTHON: '3.6'
-        PACKAGER: 'pip'
-      py35_conda:
-        VERSION_PYTHON: '3.5'
-        PACKAGER: 'conda'
-      py27_pip:
-        VERSION_PYTHON: '2.7'
-        PACKAGER: 'pip'
+      windows-py37:
+        imageName: "vs2017-win2016"
+        python.version: "3.7"
+        tox.env: py37
+      windows-py36:
+        imageName: "vs2017-win2016"
+        python.version: "3.6"
+        tox.env: py36
+      windows-py35:
+        imageName: "vs2017-win2016"
+        python.version: "3.5"
+        tox.env: py35
+      windows-py27:
+        imageName: "vs2017-win2016"
+        python.version: "2.7"
+        tox.env: py27
+      linux-py37:
+        imageName: "ubuntu-16.04"
+        python.version: "3.7"
+        tox.env: py37
+      linux-py37-high-memory:
+        imageName: "ubuntu-16.04"
+        python.version: "3.7"
+        tox.env: py37
+        RUN_MEMORY: "true"
+      linux-py36:
+        imageName: "ubuntu-16.04"
+        python.version: "3.6"
+        tox.env: py36
+      linux-py36-joblib-tests:
+        imageName: "ubuntu-16.04"
+        python.version: "3.6"
+        tox.env: "py36"
+        JOBLIB_TESTS: "true"
+      linux-py35:
+        imageName: "ubuntu-16.04"
+        python.version: "3.5"
+        tox.env: py35
+      linux-py27:
+        imageName: "ubuntu-16.04"
+        python.version: "2.7"
+        tox.env: py27
+      linux-pypy3:
+        imageName: "ubuntu-16.04"
+        python.version: "pypy3"
+        tox.env: pypy3
+        LOKY_MAX_CPU_COUNT: "2"
+
+      macos-py37:
+        imageName: "macos-10.13"
+        python.version: "3.7"
+        tox.env: py37
+      macos-py27:
+        imageName: "macos-10.13"
+        python.version: "2.7"
+        tox.env: py27
+        RUN_MEMORY: "true"
+  pool:
+    vmImage: $(imageName)
+  variables:
+    JUNITXML: 'test-data.xml'
+    LOKY_MAX_DEPTH: 5
+    COVERAGE_PROCESS_START: '.coveragerc'
+    OMP_NUM_THREADS: 4
+    PYTEST_ARGS: '-vl --timeout=60 --maxfail=5'
```

Comment:
For clarity please move the `variables` section to the top of the job declaration.